### ISSUE TITLE
Do you have a plan to support 6.6.0 version?

### ISSUE THREAD

**Issue by nuriguri** [dlumbrer/kbn_radar#8]
Hi i used your plugin well now.
But as i know, your plugin is not working on >=6.6 version.
Could you check this issue?

**Comment by dlumbrer** [comment]
Yes, I have a plan to support and continue the development. I'm a little bit busy right now, help is more than welcome!

**Comment by thomas0927** [comment]
Hi,I install your plugin on Kibana6.6.2, and got this issue:
.................
 FATAL  Error: Optimizations failure.
   5160 modules
    
    ERROR in ./plugins/kbn_radar/public/kbn_radar.js
    Module not found: Error: Can't resolve 'ui/vis/vis_category' in '/usr/mpsp/DubboServer/elk/kibana-6.6.2-linux-x86_64/plugins/kbn_radar/public'
